.github/workflows/deployment-ec2.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy on aws EC2
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: Staging AWS
+    
+    steps:
+      - uses: actions/checkout@v2 
+
+      - name: Deploy in EC2
+        env:
+            PRIVATE_KEY: ${{ secrets.AWS_EC2_PRIVATE_KEY  }}
+            HOSTNAME : ${{ secrets.AWS_EC2_PUBLIC_IP  }}
+            USER_NAME : ${{ secrets.AWS_EC2_USERNAME  }}
+            
+        run: |
+          echo "$PRIVATE_KEY" > private_key && chmod 600 private_key
+          rsync --delete -r -e "ssh -o StrictHostKeyChecking=no -i private_key" ./ ${USER_NAME}@${HOSTNAME}:~/Hex
```

Comment:
We could be more selective about what we send to ec2(e.g exclude mobile-app/ .github folders) but I don't think it's a top priority(as for https://github.com/marmelab/Hex/issues/36)